onnxruntime/core/framework/onnxruntime_typeinfo.cc

```diff
@@ -126,8 +127,20 @@ OrtStatus* OrtTypeInfo::FromOrtValue(const OrtValue& value, OrtTypeInfo** out) {
   }
 
   if (type->IsTensorSequenceType()) {
-    *out = new OrtTypeInfo(ONNX_TYPE_SEQUENCE);
-    return nullptr;
+    OrtTensorTypeAndShapeInfo* info = nullptr;
+    const auto* tensor_data_type = value.Get<onnxruntime::TensorSeq>().DataType();
+    if (tensor_data_type != nullptr) {
```

Comment:
>nsor_data_type != null [](start = 10, length = 22)

Return an error if this is nullptr ?  right now the code just falls through.  if this is a sequence isn't it supposed to have a data type?

Reply:
Added 
    else {
      return OrtApis::CreateStatus(ORT_FAIL, "OrtValue is TensorSequence type but has no element Tensor DataType associated.")
    }